src/main/java/org/aksw/iguana/cc/query/handler/QueryHandler.java

```diff
@@ -161,7 +164,10 @@ public record QueryStreamWrapper(int index, boolean cached, Supplier<InputStream
     final protected Config config;
 
     final protected QueryList queryList;
-    final protected List<QueryData> queryData;
+    protected List<QueryData> queryData;
+
+    int executableQueryCount = 0;     // stores the number of queries that can be executed
```

Comment:
mention in the docstrings of executableQueryCount and representedQueryCount when they are the same (no template or setting explicitly  template's `individualResults: true`) and how these differ for templates.